ci/builders/standalone/mac_unopt_debug_no_rbe.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
@zanderso should this be named `mac_unopt_debug_no_rbe.json` or something like `mac_clangd` since the comment says not to use it for anything else?

Reply:
That make sense to me. The name of the Linux would should probably also be changed.